plugins/verifier/schemavalidator/schemavalidation/schemavalidator.go

```diff
@@ -42,3 +43,25 @@ func Validate(schema string, content []byte) error {
 		return errors.New(e)
 	}
 }
+
+// Validates content from a byte array against a schema from a byte array
+// This is useful for testing and restricted environments as it allows loading of schemas from files
+func Validate2(schema []byte, content []byte) error {
```

Comment:
Wondering with cli test pointing to a local schema , which validate is on the code path.  Do we always call Validate schema with URL in the validator?  https://github.com/deislabs/ratify/blob/main/plugins/verifier/schemavalidator/schema_validator.go#L95

When will we switch in ValidateAgainstOfflineSchema ?

Reply:
The validator leverages a library which takes a string for the path. The path can be either a canonical file path that must start with `file://` or a URL. It does some parsing and knows how to fetch the data. It gets tricky with K8s and not being able to easily get files on the POD for testing purposes. The `file` feature of this will have to have some changes once its decided how exactly it should work. If it makes more sense to pull the validator until there are final requirements I am okay with that.

Reply:
thanks for clarification. It is good to have the file option available, I suggest If the file feature is not on code path,  we could add a link to an ratify issue so contributors can get context and track discussion/requirements there.

Reply:
Good idea, I opened #648 to track.